### PR TITLE
Hidden events: Bluespace Rift

### PR DESCRIPTION
**Pull request by FreimK**
## About The Pull Request
- Event creates 1-5 pairs of two ways teleporters in maints
- Each pair can be used 3-10 times, when usage counter becomes 0 pair disappears
- Uses sprite and description of standard teleporter

## Why It's Good For The Game
Adds diversity to possible random events.

## Changelog
:cl:
add: Added bluespace rift
/:cl:

### CODE REVIEW

File: code/game/objects/effects/portals.dm [hunk 1]
```diff
@@ -170,3 +170,36 @@ var/list/portal_cache = list()
...
+		qdel(src.target)
+		qdel(src)
+
+/obj/effect/portal/rift/teleport(atom/movable/M)
+	. = ..(M)
+	if(.)
+		var/obj/effect/portal/rift/T = target
+		usages_left -= 1
+		T.usages_left -= 1
+		close()
```
Comment (Kurgis) [inline code/game/objects/effects/portals.dm:205]: Missing newline.

**Comment by GrayRachnid** [comment]
This is literally what the wormhole event is. Except it sometimes lasts up to an hour.

**Comment by FreimK** [comment]
@MadmanMartian 
Done.

File: code/game/objects/effects/portals.dm [hunk 2]
```diff
@@ -170,3 +172,50 @@ var/list/portal_cache = list()
...
+	mask = "portal_mask"
+	failchance = 0
+	admin_announce_new = FALSE
+	var/teleportations_left
+
+/obj/effect/portal/wormhole/rift/New(loc, exit)
+	teleportations_left = rand(3, 10)
+	..(loc, 0, exit)
+	deltimer(lifetime)
+	message_admins("Bluespace rift created between [jumplink(src)] and [jumplink(src.target)]")
```
Comment (MadmanMartian) [inline code/game/objects/effects/portals.dm:192]: Maybe report how many teleportations_left there are?

Comment (FreimK) [reply code/game/objects/effects/portals.dm]: Done

**Pull request merged**